types/ovenplayer/ovenplayer-tests.ts

```diff
@@ -86,52 +88,72 @@ const providerName: string | null = player.getProviderName();
 // load(sources: OvenPlayerSource[] | OvenPlayerPlayList): void;
 player.load([
     {
-        type: "webrtc",
-        file: "file",
-        label: "myLabel",
+        type: 'webrtc',
+        file: 'file',
+        label: 'myLabel',
     },
     {
-        type: "webrtc",
-        file: "file",
+        type: 'webrtc',
+        file: 'file',
     },
 ]);
 
 player.load([
     [
         {
-            type: "webrtc",
-            file: "file",
+            type: 'webrtc',
+            file: 'file',
         },
         {
-            type: "webrtc",
-            file: "file",
+            type: 'webrtc',
+            file: 'file',
         },
     ],
     [
         {
-            type: "webrtc",
-            file: "file",
+            type: 'webrtc',
+            file: 'file',
         },
         {
-            type: "webrtc",
-            file: "file",
+            type: 'webrtc',
+            file: 'file',
         },
     ],
 ]);
 
+const quality: OvenPlayerQuality = {
+    bitrate: '4000',
+    height: 320,
+    width: 320,
+    index: '320p',
+    label: '320p',
+};
+
 // getMediaElement(): HTMLVideoElement;
 const videoElement: HTMLVideoElement = player.getMediaElement();
 
 // on(eventName: string, callback: OvenPlayerCallbackFunction): void;
-player.on("ready", (data, data2) => {
-});
+player.on('ready', (data, data2) => {});
 
 // once(eventName: string, callback: OvenPlayerCallbackFunction): void;
-player.once("play", (data) => {
-});
+player.once('play', data => {});
 
 // off(eventName: string): void;
-player.off("ready");
+player.off('ready');
 
 // remove(): void;
 player.remove();
+
+// @ts-expect-error: it's deprecated method, should throw error for newest users.
```

Comment:
i'm sorry, i'm not sure if it's "deprecated", but i mean you can't find it in [github](https://github.com/search?q=repo%3AAirenSoft%2FOvenPlayer%20setCaption&type=code) or in docs. And i where unable to run it on my projects. Property is undefined.

Reply:
Perhaps the API that was attempted to be implemented during the initial design was not implemented well and is not working now.
And right now this API is not being used at all. I think the deprecated method is correct.